FFmpegInterop/MediaSampleProvider.cpp

```diff
@@ -228,7 +228,7 @@ HRESULT MediaSampleProvider::GetNextPacket(AVPacket** avPacket, LONGLONG & packe
 		*avPacket = packet;
 
 		packetDuration = packet->duration;
-		if (packet->pts != AV_NOPTS_VALUE)
+		if (packet->pts != AV_NOPTS_VALUE && packet->pts >= 0 && packet->pts - m_nextPacketPts < 600000000)
```

Comment:
Remove pts >= 0 check (same reason as in UncompressedSampleProvider comment).

And then I thing I have spotted a problem in MediaSampleProvider (not related to your change): The m_nextPacketPts never gets reset on a seek operation. After a seek, the m_nextPacketPts will stay on the same value as before, so we might use wrong pts values if packets after seek do not contain a pts value on their own. To fix this, we'd need to update m_nextPacketPts with the seek target position (taking into account). It probably only ever affects broken files, still, I think we should fix this. It might improve seeking in partly broken files. This does not need to be fixed in this PR, we can do a separate PR for that.